client/my-sites/themes/theme-options.js

```diff
@@ -39,10 +39,18 @@ import { FEATURE_UNLIMITED_PREMIUM_THEMES } from 'lib/plans/constants';
 
 const purchase = config.isEnabled( 'upgrades/checkout' )
 	? {
-		label: i18n.translate( 'Purchase', {
-			context: 'verb'
-		} ),
-		extendedLabel: i18n.translate( 'Purchase this design' ),
+		label: ( state, siteId ) => config.isEnabled( 'jetpack/pijp' ) && isJetpackSite( state, siteId )
+			? i18n.translate( 'Upgrade to activate', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Upgrade and activate' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).